index.html

```diff
@@ -15,20 +15,20 @@
         </div>
 
         <ul class="nav-links">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
+          <li><a href="#home">Portfolio</a></li>
+          <li><a href="#about-me">About</a></li>
           <li><a href="#contact">Contact</a></li>
           <a href="#"><ion-icon name="mail-outline"></ion-icon></a>
         </ul>
 
-        <div class="vector">
+        <div class="vectors">
           <span></span>
           <span></span>
           <span></span>
         </div>
       </nav>
 
-      <section class="portfolio">
+      <section class="portfolio" id="home">
```

Comment:
- [ ] **[OPTIONAL]** I think you may try to add a smooth scroll effect to a page in CSS, you can use the **[scroll-behavior](https://www.w3schools.com/cssref/pr_scroll-behavior.php)** property. This property allows you to specify whether the scroll position should smoothly animate to a new position, rather than immediately jumping to it. 😉😉

Here's an example:

```
body {
  scroll-behavior: smooth;
}
```